flyway-core/src/main/java/org/flywaydb/core/internal/command/DbMigrate.java

```diff
@@ -204,11 +204,19 @@ private Integer migrateGroup(boolean firstRun) {
             currentSchemaVersion = infoService.current().getVersion();
         }
         if (firstRun) {
-            LOG.info("Current version of schema " + schema + ": " + currentSchemaVersion);
-
-            if (configuration.isOutOfOrder()) {
-                LOG.warn("outOfOrder mode is active. Migration of schema " + schema + " may not be reproducible.");
+            if (schema.getName() != "null") {
```

Comment:
illegal equals check for strings

Reply:
@andrevanzuydam ?

Reply:
I'm open to suggestions, I would need to build the project again to see what the implications were, Firebird does not have schemas